src/button/constants.js

```diff
@@ -20,7 +20,9 @@ export const SHAPE = {
 };
 
 export const SIZE = {
+  medium: 'default',
   default: 'default',
   compact: 'compact',
+  small: 'compact',
```

Comment:
Same here. Saying that I'm inclined toward not introducing this change and stick to the current naming.

Reply:
```suggestion
```